From 99aed1cccdb8a5b26b9287e6e60c65d4d56c7f8e Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 16 Mar 2022 16:26:06 +0100
Subject: [PATCH 8/9] Revert "alt_core.c: Add potentially missing assignment of
 p->on_cpu in sched_fork"

This reverts commit f6e10a04d0eab1ad33ac82ef36dd772baeb0a34c.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 kernel/sched/alt_core.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/kernel/sched/alt_core.c b/kernel/sched/alt_core.c
index 4d7a4e1e5..43aece4db 100644
--- a/kernel/sched/alt_core.c
+++ b/kernel/sched/alt_core.c
@@ -3056,9 +3056,6 @@ int sched_fork(unsigned long clone_flags, struct task_struct *p)
 #ifdef CONFIG_SCHED_INFO
 	if (unlikely(sched_info_on()))
 		memset(&p->sched_info, 0, sizeof(p->sched_info));
-#endif
-#if defined(CONFIG_SMP)
-	p->on_cpu = 0;
 #endif
 	init_task_preempt_count(p);
 
-- 
2.35.1.354.g715d08a9e5

